modules/machine-user-infra-machines-ibm-z.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc
+
+:_content-type: PROCEDURE
+[id="machine-user-infra-machines-ibm-z_{context}"]
+= Creating {op-system} machines on {ibmzProductName} with z/VM
+
+You can create more {op-system-first} compute machines running on {ibmzProductName} with z/VM and attach them to your existing bare metal x86_64 cluster.
+
+.Prerequisites
+
+* A domain name server (DNS) that can perform hostname and reverse lookup for the nodes.
+* An HTTP or HTTPS server running on your provisioning machine that is accessible to the machines you create.
+
+.Procedure
+
+. Extract the Ignition config file from the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc extract -n openshift-machine-api secret/worker-user-data-managed --keys=userData --to=- > worker.ign
+----
+
+. Upload the `worker.ign` Ignition config file you exported from your cluster to your HTTP server. Note the URL of this file.
+
+. You can validate that the Ignition file is available on the URLs. The following example gets the Ignition config file for the compute node:
++
+[source,terminal]
+----
+$ curl -k http://<HTTP_server>/worker.ign
+----
+
+. Download the {op-system-base} live kernel, initramfs, and rootfs files by running to following commands:
++
+[source,terminal]
+----
+curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.kernel.location')
+----
++
+[source,terminal]
+----
+curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.initramfs.location')
+----
++
+[source,terminal]
+----
+curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
```

Comment:
should have a $ in front (if this is the consistent style of writing commands executed on a shell)